_data/templates.json

```diff
@@ -1463,9 +1506,51 @@
 					"Veuillez suivre les instructions d'implémentation sous l'onglet «&nbsp;Standard&nbsp;»\"."
 				]
 			}
+		},
+		{
+			"@id": "_:implement_topic_stackedheader",
+			"iteration": "_:iteration_topic_4",
+			"name": {
+				"en": "Topic page with stacked header",
+				"fr": "Page de sujet avec en-tête superposée"
```

Comment:
Can you please remove the extra "e" at the end of the following sentence: "Titre du sujet sans en-tête superposée"